packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -5,7 +5,7 @@ import { ItemType } from './item-type';
 
 export declare type ReportFormat = 'sarif';
 export declare type ScanState = 'unknown' | 'pass' | 'fail';
-export declare type OnDemandPageScanRunState = 'unknown' | 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
+export declare type OnDemandPageScanRunState = 'pending' | 'not found' | 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
```

Comment:
Please revert all changes related to 'pending' This is will included in my PR

Reply:
There's a lot of confusion been happening, in the meeting I was asked to fix this, not sure why you will do it or what's the point of removing such a small change so you can repeat it, but I'll do it for the sake of getting this closed because the review it taking way more time than it should be.

Reply:
As per our offline conversation yesterday I  am working on web api controllers rest api response codes and scan run result format fix. This will have changes related to all web api controllers and include state result update. This is what I was meant by asking to not do any changes related to state yesterday. Can you please update only time (see above open comment) for 'too late response'. Thank you

Reply:
As per our offline conversation, I told you I already did it after you asked because I was asked in the meeting to do it, and I told feel free to add what you want to the "PR", there's a lot of confusion needs to be addressed before I fix anymore comments